src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(getRandomBall());
```

Comment:
Avoid calling static methods from the `Lottery` class. Instead, create an instance of `Lottery` and call the instance method to get a random ball.